pkg/logs/sender/compressor.go

```diff
@@ -0,0 +1,46 @@
+package sender
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
+	"github.com/DataDog/datadog-agent/comp/serializer/compression"
+)
+
+type Compressor struct {
+	compression compression.Component
+}
+
+var (
+	tlmTotalPayloads = telemetry.NewCounter("logscompressor", "total_payloads",
```

Comment:
Is there a better name for these metrics? `compressor.total_payloads` is already taken up by the metrics compressor...

Reply:
They aren't needed, so I've removed them..